propelauth_py/__init__.py

```diff
@@ -1,164 +1,329 @@
 from collections import namedtuple
 
-from propelauth_py.api import _fetch_token_verification_metadata, TokenVerificationMetadata, \
-    _fetch_user_metadata_by_user_id, \
-    _fetch_user_metadata_by_email, _fetch_user_metadata_by_username, _fetch_batch_user_metadata_by_user_ids, \
-    _fetch_batch_user_metadata_by_emails, _fetch_batch_user_metadata_by_usernames, OrgQueryOrderBy, UserQueryOrderBy, \
-    _fetch_org, _fetch_org_by_query, _fetch_users_by_query, _fetch_users_in_org, _create_user, _update_user_email, \
-    _update_user_metadata, _create_magic_link, _migrate_user_from_external_source, _create_org, _update_org_metadata, \
-    _add_user_to_org, _delete_user, _disable_user, _enable_user, _allow_org_to_setup_saml_connection, \
-    _disallow_org_to_setup_saml_connection, _update_user_password, _create_access_token, _disable_user_2fa, \
-    _enable_user_can_create_orgs, _disable_user_can_create_orgs, _fetch_api_key, \
-    _fetch_current_api_keys, _fetch_archived_api_keys, _create_api_key, \
-    _update_api_key, _delete_api_key, _validate_api_key, _validate_personal_api_key, _validate_org_api_key
-from propelauth_py.auth_fns import wrap_validate_access_token_and_get_user, \
-    wrap_validate_access_token_and_get_user_with_org, \
-    wrap_validate_access_token_and_get_user_with_org_by_minimum_role, \
-    wrap_validate_access_token_and_get_user_with_org_by_exact_role, \
-    wrap_validate_access_token_and_get_user_with_org_by_permission, \
-    wrap_validate_access_token_and_get_user_with_org_by_all_permissions, \
-    validate_org_access_and_get_org_member_info, \
-    validate_minimum_org_role_and_get_org, \
-    validate_exact_org_role_and_get_org, \
-    validate_permission_and_get_org, \
-    validate_all_permissions_and_get_org
+from propelauth_py.api.user import (
+    _fetch_user_metadata_by_user_id,
+    _fetch_user_metadata_by_email,
+    _fetch_user_metadata_by_username,
+    _fetch_batch_user_metadata_by_user_ids,
+    _fetch_batch_user_metadata_by_emails,
+    _fetch_batch_user_metadata_by_usernames,
+    _fetch_users_by_query,
+    _fetch_users_in_org,
+    _create_user,
+    _update_user_email,
+    _update_user_metadata,
+    _delete_user,
+    _disable_user,
+    _enable_user,
+    _update_user_password,
+    _disable_user_2fa,
+    _enable_user_can_create_orgs,
+    _disable_user_can_create_orgs,
+    _validate_personal_api_key,
+)
+from propelauth_py.api.org import (
+    _fetch_org,
+    _fetch_org_by_query,
+    _create_org,
+    _remove_user_from_org,
+    _update_org_metadata,
+    _add_user_to_org,
+    _allow_org_to_setup_saml_connection,
+    _disallow_org_to_setup_saml_connection,
+    _validate_org_api_key,
+)
+from propelauth_py.api.magic_link import _create_magic_link
+from propelauth_py.api.token_verification_metadata import (
+    _fetch_token_verification_metadata,
+    TokenVerificationMetadata,
```

Comment:
I believe this import needs to stay in propelauth_py.api for backwards compability

Reply:
Sure, makes sense -- will add an import statement there as well

Reply:
Updated to declare it in the `propelauth_py/api/__init__.py` file to avoid a circular dependency. The import will now be `from propelauth_py.api import TokenVerificationMetadata`